apps/crn-server/src/autogenerated-gql/graphql.ts

```diff
@@ -3696,12 +3696,12 @@ export type ResearchOutputsDataSharingStatusInputDto = {
   iv: Maybe<Scalars['String']>;
 };
 
-/** The structure of the Subtype field of the Research Outputs content type. */
+/** The structure of the Subtype (deprecated) field of the Research Outputs content type. */
```

Comment:
Can we remove the subtype from the schema?

Reply:
Maybe in next PR when we are going to be sure that everything is working correctly?